Packs/MajorBreachesInvestigationandResponse/ReleaseNotes/1_6_31.md

```diff
@@ -0,0 +1,71 @@
+
+#### Playbooks
+
+##### CVE-2021-22893 - Pulse Connect Secure RCE
```

Comment:
Let's talk about it.  In this case the release note should include only a description and info about changes.